src/app/mat-main-nav/mat-main-nav.component.scss

```diff
@@ -0,0 +1,63 @@
+@import "@angular/material/core/style/_variables";
+
+.mat-toolbar-row,
+.mat-toolbar-single-row {
+  @media screen and (#{$mat-xsmall}) {
+    height: 72px;
+  }
+}
+
+.mat-icon-button.logo-icon {
+  width: 100px;
+  height: 40px;
+  line-height: 1;
+}
+
+.sidenav-container {
+  height: 100%;
+}
+
+.sidenav {
+  width: 90%;
+}
+
+.icon_nhs_logo.mat-icon {
+  width: 100px;
+  height: 40px;
+
+  &:hover {
+    box-shadow: 0 0 0 4px #003d78;
```

Comment:
this colour would benefit from being in a variable

Reply:
done